pkgs/by-name/ro/ron-lsp/package.nix

```
@@ -0,0 +1,35 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  nix-update-script,
+}:
+
+rustPlatform.buildRustPackage (finalAttrs: {
+  pname = "ron-lsp";
+  version = "0.1.1";
+
+  # https://github.com/jasonjmcghee/ron-lsp
+  src = fetchFromGitHub {
+    owner = "jasonjmcghee";
+    repo = "ron-lsp";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-RHQ98+6cLODVGWHRb8/Ved5ipk5imXW4e/pI2ms2Cq0=";
```

```suggestion
    hash = "sha256-c+cJrXINuoK+NR1rMSrOeZqZzrEcg/brSTKSTu5mNr4=";
```

This hash seems to be wrong. The fetchers will not re-download the path if already present in your store so it may be that it looked like it was working on your end.